.travis-build.sh

```diff
@@ -71,13 +75,13 @@ if [[ "${GROUP}" == "downstream" || "${GROUP}" == "all" ]]; then
   ##  * daikon-typecheck: (takes 2 hours)
 
   # checker-framework-inference: 18 minutes
-  (cd .. && git clone --depth 1 https://github.com/typetools/checker-framework-inference.git)
+  (cd .. && git clone --depth 1 https://github.com/${SLUGOWNER}/checker-framework-inference.git)
```

Comment:
`${SLUGOWNER}/checker-framework-inference`may not exist.  If it doesn't typetools should be used.  (Same comment for `plume-lib`.